### PR TITLE
Clarify volume checking

### PR DESCRIPTION
**Pull request by aragilar**
This adds an additional logging statement to specify which file is being
looked for, and corrects two bits of documentation.

### CODE REVIEW

**Comment by rtobar** [comment]
HI @aragilar, thanks for the PR!

In general it looks alright, but could I ask you to re-open this against the `master` branch? I'd rather restrict commits on the stable branches only to bug fixes and not to general improvements, and this belongs to the latter category. So a heads up that I'll be closing this PR.

Regardless of that, I'll also add a couple of comments on the patch, but again without the intention to finally merge as part of this PR.

If for some reason it becomes a bit difficult for you to apply the same patch on the `master` (can't remember all the things that have changed since v11 was spawned...) let me know and I'll be happy to help with that.

File: src/ngamsPlugIns/ngamsPlugIns/mwa/ngamsMWAOnlinePlugIn.py [hunk 1]
```diff
@@ -180,4 +180,4 @@ def _restoreSubscriptionInfoFromDisk(srvObj):
     except Exception:
         logger.exception('Fail to append filelist to subscription info list')
     finally:
-        srvObj._subscriptionSem.release()
\ No newline at end of file
+        srvObj._subscriptionSem.release()
```
Comment (rtobar) [inline src/ngamsPlugIns/ngamsPlugIns/mwa/ngamsMWAOnlinePlugIn.py]: Could this change be left out? I'm sure it was unintentional. There is a similar one in the other modified file.

File: src/ngamsPlugIns/ngamsPlugIns/ngamsGenericOnlinePlugIn.py [hunk 2]
```diff
@@ -51,6 +51,7 @@
 import glob
 import logging
 import os
+import os.path
```
Comment (rtobar) [inline src/ngamsPlugIns/ngamsPlugIns/ngamsGenericOnlinePlugIn.py]: Not necessary, I believe

File: src/ngamsPlugIns/ngamsPlugIns/ngamsGenericOnlinePlugIn.py [hunk 3]
```diff
@@ -133,5 +133,7 @@ class (ngamsServer).
                                        setManufacturer(manufact).\
                                        setDiskId(diskId).\
                                        setDeviceName(devName)
+        else:
+            logger.debug("%s does not exist", volInfoFile)
 
-    return diskInfoDic
\ No newline at end of file
+    return diskInfoDic
```
Comment (rtobar) [inline src/ngamsPlugIns/ngamsPlugIns/ngamsGenericOnlinePlugIn.py]: As above, please leave out as it's probably unintended

**Comment by aragilar** [comment]
Sorry, I'll fix this and create a new PR.

**Pull request closed without merging**